documentation/book/appendix_metrics.adoc

```diff
@@ -41,10 +41,26 @@ https://prometheus.io/[Prometheus] is an open-source set of components for syste
 
 To deploy the Prometheus Operator in your Kafka cluster, apply the `.yaml` files from the https://github.com/coreos/prometheus-operator/tree/master/example/rbac/prometheus-operator[Prometheus CoreOS repository].
 
-To use a different namespace than that specified in the repository files, use the following command to download and edit the `prometheus-operator-deployment.yaml` file:
+To use a different namespace than that specified in the repository files, use the following commands to download and edit the files from respository:
 
 [source,shell,subs="+quotes"]  
 curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-deployment.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/;s/regex: myproject/regex: _my-namespace_/' > prometheus-operator-deployment.yaml
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-cluster-role-binding.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/;s/regex: myproject/regex: _my-namespace_/' > prometheus-operator-cluster-role-binding.yaml
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-service-account.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/;s/regex: myproject/regex: _my-namespace_/' > prometheus-operator-service-account.yaml
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-cluster-role.yaml
+
+It might happen you need to remove `securityContext` from Prometheus Operator Deployment which is located in the `prometheus-operator-deployment.yaml` file.
```

Comment:
Please reword as follows:

If needed, you can remove the `securityContext` from the Prometheus Operator Deployment. You can either manually remove the `spec.template.spec.securityContext` property from the `prometheus-operator-deployment.yaml` file,  or run the following command: